setup/kubernetes/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Kubernetes
+description: Learn how to set up a K8 cluster for your Coder deployment.
+---
+
+You can install and deploy Coder onto any Kubernetes cluster (version 1.13.7+)
+that meets the [requirements](../requirements.md).
+
+To help you get up and running, Coder offers the following deployment guides:
+
+- [Amazon Elastic Kubernetes Engine](../kubernetes/aws.md)
```

Comment:
Instead of having the bullet points here you can use:
```
<children></children>
```
to display all child topics.